src/api/cvc4cpp.h

```diff
@@ -2559,6 +2559,15 @@ class CVC4_PUBLIC Solver
   Term mkTuple(const std::vector<Sort>& sorts,
                const std::vector<Term>& terms) const;
 
+  /**
```

Comment:
I think we shouldn't include this yet either.

Reply:
Done